modules/passkey/hardhat.config.ts

```diff
@@ -58,7 +59,7 @@ const config: HardhatUserConfig = {
             enabled: true,
             runs: 10_000_000,
           },
-          viaIR: false,
+          viaIR: true,
```

Comment:
This has some nasty implications for the FCL lib AFAIR. I would not sneak in this change in this PR.

Reply:
Oops, sorry. Somehow, it made its way after my benchmarking.